queue.go

```diff
@@ -533,3 +560,10 @@ func (queue *redisQueue) ensureConsuming() error {
 		return nil
 	}
 }
+
+// jitteredDuration calculates and returns a value that is +/-10% the input duration
+func jitteredDuration(duration time.Duration) time.Duration {
+	factor := float64(duration) * randSrc.Float64() * 0.2 // a jitter factor of +/-10%
+	base := float64(duration) * 0.9
+	return time.Duration(factor + base)
+}
```

Comment:
```suggestion
func jitteredDuration(duration time.Duration) time.Duration {
	base := 0.9
	factor := randSrc.Float64() * 0.2 // a jitter factor of +/-10%
	return time.Duration(float64(duration) * (factor + base))
}
```

Is this more readable?

Reply:
```suggestion
func jitteredDuration(duration time.Duration) time.Duration {
	factor := 0.9 + randSrc.Float64() * 0.2 // a jitter factor between 0.9 and 1.1 (+-10%)
	return time.Duration(float64(duration) * factor)
}
```
I think `factor` should be the thing you multiply with.